.github/workflows/ci-documentbot.yml

```diff
@@ -0,0 +1,27 @@
+name: Auto Labeling
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - labeled
+
+  
+
+# A GitHub token created for a PR coming from a fork doesn't have
+# 'admin' or 'write' permission (which is required to add labels)
+# To avoid this issue, you can use the `scheduled` event and run
+# this action on a certain interval.And check the label about the
+# document.
+
+jobs:
+  labeling:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: pr-bot-test/auto-labeling@master
```

Comment:
I believe that is it not allowed in the ASF to use third party GH Actions using branch name references.
Can we copy this code into an ASF repository ? or at least point to a specific SHA (if it is possible)

Reply:
Can we put this action into https://github.com/apache/pulsar-test-infra ?

Reply:
> I believe that is it not allowed in the ASF to use third party GH Actions using branch name references.
> Can we copy this code into an ASF repository ? or at least point to a specific SHA (if it is possible)

I have fixed the problem of GH action using third-party branches. I changed to point a sha to quote script. Do you think this is OK? Or can you tell me where the ASF repo is? I'll push the script in.

Reply:
this is the repo
https://github.com/apache/pulsar-test-infra

having it in our repo is better, as private repos may disappear some day

Reply:
Thank you. I have submitted a pullrequest in this repo. When the merge succeeds, I will apply for review again.

Reply:
I have changed the repo in this GH Actions. 
PTAL, thanks.